src/black/strings.py

```diff
@@ -18,6 +20,14 @@
     r"^([" + STRING_PREFIX_CHARS + r"]*)(.*)$", re.DOTALL
 )
 FIRST_NON_WHITESPACE_RE: Final = re.compile(r"\s*\t+\s*(\S)")
+UNICODE_RE = re.compile(
```

Comment:
Should probably also make this Final.

And using `re.MULTILINE` might be nice so you can put comments inside the regex.